README.md

```diff
@@ -1,2 +1,22 @@
 # comp110-worksheet-C
 Base repository for COMP110 worksheet C
+
+(a)go through two lists then see if the list are different sizes but have the same final number
+
+(b)N has to go through two loops which causes it to have an O(n^2)
+
+(c)changing line 4 will always make the first condition true and dosn't make the second condition false
+
+(d)N only has to go through one loop
+
+(e) no as i said above N only has to go through one loop
```

Comment:
O(n^2 / 2) = O(n^2) (ignoring the leading constant), so in fact the algorithm is still quadratic.